### ISSUE TITLE
[Dataset request] Zr_Sn_2023

### ISSUE THREAD

**Issue by jvita** [colabfit/data-lake#24]
### Name

Josh Vita

### Email

vita1@llnl.gov

### Dataset name

Zr_Sn_2023

### Authors

Haojie Mei, Liang Chen, Feifei Wang, Guisen Liu, Jing Hu, Weitong Lin, Yao Shen, Jinfu Li, Lingti Kong

### Links

* Paper: https://www.sciencedirect.com/science/article/pii/S0022311523005615
* Data: unknown; email author?

### Dataset description

[[AI-generated description]]

- The dataset contains data from density functional theory calculations of various atomic configurations of pure Zr, pure Sn, and Zr-Sn alloys with different structures, defects, and compositions.
- The dataset includes energies, forces, and stresses for each configuration, which are used as target quantities for fitting the potential parameters.
- [The dataset is randomly divided into a training set and a validation set, comprising 80% and 20% of the total number of configurations, respectively.
- The dataset contains data corresponding to 23,956 configurations in total, with 723,497 force entries and 143,736 stress entries.

Source: Conversation with Bing, 10/26/2023

### Method

DFT

### Software

VASP

### Property types

Atomic forces, Cauchy stress, Potential energy

### Elements

Zr, Sn

### Permissions

- [ ] I confirm that I have the necessary permissions to submit this dataset

**Comment by gpwolfe** [comment]
data has been requested from corresponding author

**Comment by gpwolfe** [comment]
author has responded, staged for ingest following next database update

**Comment by gpwolfe** [comment]
new database now live
